package.json

```diff
@@ -173,7 +174,6 @@
     "react-native-swipe-gestures": "^1.0.2",
     "react-native-swipe-list-view": "^3.2.5",
     "react-native-tcp": "git://github.com/EdgeApp/react-native-tcp.git#rn62",
-    "react-native-ticker": "3.2.0",
```

Comment:
Why is this repo removed? Isn't this required?

Reply:
I cloned your branch and tried to build and run but I definitely got the missing library error. I added this back and rebuilt and the app runs but I get the following error

<img width="531" alt="Screen Shot 2021-01-19 at 23 50 11" src="https://user-images.githubusercontent.com/6079354/105143979-7135e200-5ab1-11eb-8d82-556f73ad4065.png">

Reply:
Thanks for taking the time to check this :) 
In retrospect I should have put this up as a Draft PR as this is indeed not ready for prime time. I ran into that error as well. 

To address where the first question: 
Repo was removed because it was referencing the repo in the npm artifactory not our forked repo. 

I pointed the react-native-ticker to the following to fix some things locally: 
git://github.com/j0shwheeler/react-native-ticker#build_fixes

In order to fix that react-native ticker issue. There are two known issues with this repository that popped up while testing that I need to fix: 
1) First issue, that measurement.height property. The value measurement object is being determined dynamically by the ticker library based on its child components and it is buggy. I am going to patch our forked repo in order to resolve this
2) Second issue, Running into build issues when importing react-native-ticker into edge-react-gui. It is a typescript project that compiles properly when it is published to npm. Though compiling it locally when including it in the project is causing an issue. I've tried compiling it in a few different ways though there were different issues with each method. Easiest way to resolve this I think would be to actually publish this repo to npm with our own edge namespace. Though I think that I can still get it to work having to publish it just need to fiddle with it a bit more. 

Once those issues are completed and the react-native-ticker is back in action I will be able to clean up that ThemedTicker component and get it in all the places we want and finally do a manual QA on iOS and Android and request a final review!! 

@paullinator

Reply:
Quick update (1) is fixed. The UI now does not error when measurement is undefined. It waits for the height and width properties to be available before it attempts to render using those values now. 
https://github.com/EdgeApp/react-native-ticker/pull/3/files (still WIP)

Next opportunity I have this week, I will fix the build issues and then just keep pressing forward. 
The new UI looks awesome btw

@paullinator

Reply:
Great to hear. Happen to have any video demos you can share?

Reply:
Yep! Here is a screen recording. 
![UI2RotatingNums](https://user-images.githubusercontent.com/5866571/105868456-0c93ef00-5fab-11eb-98a0-cae07e2998c7.gif)

Reply:
looks great !

Reply:
Thanks, 
Made more progress today. 

All of the implementation code in the PR should be good to go pending review. 
Testing on my iOS simulator checks out. 

Still need to write unit tests and try this app out on a real android and iOS device. 

I made the numbers spin only for the large numbers on top of the wallet list and transaction list screens. 

The other smaller numbers had issues so did not convert those. 

The screenshot in this comment shows what happens when the EdgeText is converting to a spinning number for the numbers on the individual wallets. ThemedTicker does not support the adjustFontSizeToFit property. 
![Screen Shot 2021-02-01 at 7 14 44 PM](https://user-images.githubusercontent.com/5866571/106553215-c7514f00-64cd-11eb-8aba-b497f47f615d.png)

That said, its getting closer. 

One question - 
Do you want the .lock files included in the commits or should they be removed? 

Ready for feedback @paullinator

Reply:
Hi Josh. Awesome to hear things are looking good. Definitely commit the lock files.I will take a look at the PR this week